_posts/2018-05-24-open-source-hackillinois-2018.markdown

```diff
@@ -0,0 +1,45 @@
+categories: recap
+tags: HackIllinois2018, featured
+image: /assets/article_images/open-source-hackillinois-2018/panel.jpg
+author: Rohin Bhasin
+---
+
+Open Source carried a huge presence at HackIllinois 2018. This year, we had over [40 mentors](https://2018.hackillinois.org/mentors) and [25 sponsors](2018.hackillinois.org) at HackIllinois, and everyone came eager to contribute to the Open Source community! The goal of HackIllinois has expanded in recent years to connect with the developer community by encouraging students and hackers from all over the country to contribute to the tools they use every day. Hackers get flexibility as to what they want to work on - they can opt to build their own project, or they can have the unique experience of learning how to contribute to existing Open Source software. To facilitate these contributions, the HackIllinois Team invites mentors from established Open Source projects to work with Hackers at the event and help them get involved.
+
+# Mentors
+
+Open Source contributions at HackIllinois would not be possible without the many amazing mentors that attended the event this year. Mentorship is essential to the HackIllinois experience: these veteran Open Source contributors and engineers help students get the ball rolling on contributing to Open Source. 
+
+To initially introduce students to the mentors, we held panels regarding Web/Data and Systems/Miscellaneous so students could come hear about the fields they were interested in. Afterward, mentors briefly described their projects and students had a chance to discuss with mentors and join their teams.
+
+![Mentors with experience in Web/Data discuss at the panel in Siebel](/assets/article_images/open-source-hackillinois-2018/panel_speakers.jpg "panel")
+
+​	Afterward, hacking began! Students began learning the ropes of Open Source and finding issues to solve. Many mentors had tagged issues for HackIllinois beforehand and were prepared to guide students through the initially intimidating process of understanding and improving a codebase.
+
+![Students start to work with mentor Andy Schwartzmeyer on the Apache Mesos project](/assets/article_images/open-source-hackillinois-2018/powershell.jpg "mesos")
+
+Last year, hacker Trent Spice worked with the Rust team and [won the Long Con prize](https://blog.hackillinois.org/spotlight/2017/09/29/open-Source-at-hackillinois-rust-cookbook.html) for HackIllinois 2017. This year, he came back with other students from Indiana University–Purdue University Indianapolis (IUPUI) and continued to work on Rust with mentors Steve Klabnik and Ashley Williams. HackIllinois aims to build these long-term relationships between students and the Open Source community, and our staff is super glad that groups from HackIllinois are still working together to contribute to and build on projects like [Rust Cookbook](https://github.com/rust-lang-nursery/rust-cookbook) and [RustBridge](https://github.com/rustbridge).
```

Comment:
Link to Steve and Ashley somewhere on the Internet (either personal pages or GitHub, up to you)